client/task-list/tasks/products/products.js

```diff
@@ -16,12 +17,19 @@ import ProductTemplateModal from './product-template-modal';
 const subTasks = [
 	{
 		key: 'addProductTemplate',
-		title: __( 'Start with a template (recommended)', 'woocommerce-admin' ),
+		title: (
+			<>
+				{ __( 'Start with a template', 'woocommerce-admin' ) }
+				<span className="task-list-recommended-label">
```

Comment:
There is a `Pill` component in `@woocommerce/components` that you might be able to use, see `client/profile-wizard/steps/product-types.js` for a usage (the product type selection step in the onboarding wizard)

Reply:
Ooh thanks @becdetat, I didn't know about this. I tried to look for one for a little while, and couldn't find any examples either. I guess `Pill` does make sense for the name, it's tricky sometimes 🙈

Reply:
@becdetat fixed in: https://github.com/woocommerce/woocommerce-admin/pull/6177/commits/94efc8040fb76634f20216c0526f69d15b41f3df